lib/server.js

```diff
@@ -7,11 +7,12 @@ var Promise = require('bluebird');
 var format = require('util').format;
```

Comment:
are `format` unused now?

Reply:
Yes, I switched to the more reliable `require('url').format` and forgot to remove the `format` import. I've seen the Travis error: I'll push an update.